packages/frontend/web-ui/src/scss/auth/register/_page.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../base/variables' as var;
+
+.register-form-grid {
+  background-color: var(--cg-text-primary);
+  border-radius: 10px;
+  text-align: center;
+  color: var(--cg-bkg-primary-light);
+  padding: 3rem;
+  margin: 0rem 2rem;
+}
+
+@media (min-width: var.$sm) {
+  .register-form-grid {
+    margin: 0;
+    max-width: 450px;
+  }
+}
+
+.register-title-text-position {
```

Comment:
Same issue than it's equivalent class in the login page.

Reply:
Done! I've renamed the CSS class name.